webapp/crashstats/libbugzilla.py

```diff
@@ -80,40 +106,55 @@ def crash_report_to_description(crash_report_url, processed_crash):
         lines.append("")
         lines.append(f"Reason: ```{processed_crash['reason']}```")
 
-    # If there's a java_stack_trace, add that; this is a big string blob, so we truncate
-    # it at 5,000 characters but don't otherwise format it
-    if processed_crash.get("java_stack_trace"):
+    frames = None
+    if "json_dump" in processed_crash:
+        # Generate frames from the stackwalker output from parsing a minidump
+        threads = processed_crash.get("json_dump", {}).get("threads")
+        if threads:
```

Comment:
this seems like exactly what `:=` is for:
```suggestion
        if threads := processed_crash.get("json_dump", {}).get("threads"):
```

Reply:
I almost never use `:=` for whatever reason, but yes--this would work here. I can change it.

Reply:
In other parts of the codebase, we use `glom` to deeply extract values from the structures, but I didn't want to use that here and introduce another dependency for this module. But ... it sure would read better if I rolled my own `glom` for this module. I'll do that.